GeneratingDataset.py

```diff
@@ -1051,16 +1078,65 @@ def get_audio_features(self, audio, sample_rate):
       assert feature_data.shape[1] == self.get_feature_dimension()
 
     if self.norm_mean is not None:
-      feature_data -= self.norm_mean[None, :]
+      if isinstance(self.norm_mean, (int, float)):
+        feature_data -= self.norm_mean
+      else:
+        feature_data -= self.norm_mean[None, :]
+
     if self.norm_std_dev is not None:
-      feature_data /= self.norm_std_dev[None, :]
+      if isinstance(self.norm_std_dev, (int, float)):
+        feature_data /= self.norm_std_dev
+      else:
+        feature_data /= self.norm_std_dev[None, :]
+
+    if self.join_frames is not None:
+      pad_len = self.join_frames - (feature_data.shape[0] % self.join_frames)
+      pad_len = pad_len % self.join_frames
+      new_len = feature_data.shape[0] + pad_len
+      feature_data = numpy.pad(feature_data, pad_width=((0, pad_len), (0, 0)), mode="edge")
+      feature_data = numpy.reshape(feature_data,
+                                   newshape=(int(new_len/self.join_frames), feature_data.shape[1]*self.join_frames),
```

Comment:
There are some spaces missing (around any operators).

Reply:
Also, never use `/` for ints. Use `//` instead. And then you also don't need the `int` cast.

Reply:
When being a parameter no spaces are PEP8 conform so there was no error, but I can change it.

Reply:
I think the code should be consistent. Also, even if not in PEP8, I think this is standard Python code convention.